### PR TITLE
BaSP-TG-73: Fix stylesheets

### PR DESCRIPTION
**Pull request by Ansitox**
Fix and clear some css files

### CODE REVIEW

File: src/Components/Shared/Button/button.module.css [hunk 1]
```diff
@@ -4,6 +4,8 @@
     font-size: 16px;
     padding: 10px 30px;
     text-align: center;
+    border: none;
+    font-weight: 600;
```
Comment (CristianBucci) [inline src/Components/Shared/Button/button.module.css:8]: The order should be:
    padding: 10px 30px;
    border: none;
    font-size: 16px;
    font-weight: 600;
    text-align: center;
    background-color: #FF0000;
    color: white;

File: src/Components/Shared/Button/button.module.css [hunk 2]
```diff
@@ -12,4 +14,6 @@
...
```
Comment (CristianBucci) [inline src/Components/Shared/Button/button.module.css:18]: Same as above!

**Review by CristianBucci — APPROVED** [review]
Everything look fine!

File: src/Components/Shared/Button/button.module.css (hunk 1 above)
Comment (Ansitox) [reply src/Components/Shared/Button/button.module.css:8]: done

File: src/Components/Shared/Button/button.module.css (hunk 2 above)
Comment (Ansitox) [reply src/Components/Shared/Button/button.module.css:18]: done

**Review by renzopimentel — APPROVED** [review]
Perfect

**Pull request merged**